### ISSUE TITLE
sphereface

### ISSUE THREAD

**Issue by MartinDione** [deepinsight/insightface#5]
What did you mean by sphereface? is it Asoftmax or L-Margin or Isoftmax ? because I think it's not same as Asoftmax ... 
If yes did you test it and get similar behavior to the original one

**Comment by nttstar** [comment]
I just normalized the weight and then use L-Margin(L-Softmax).  It equals to ASoftmax logically.

**Comment by MartinDione** [comment]
Just I want to test combine both of ASoftmax and Triplet... did you test it before?
Thanks

**Comment by nttstar** [comment]
Yes it works. You need to train ASoftmax firstly and finetune it by triplet with semi-hard mining.
It brings +0.2 acc on my previous megaface top1 test.

**Comment by MartinDione** [comment]
What is your highest accuracy you have got on Megaface, and which loss did you use?

**Comment by nttstar** [comment]
We will publish all experiment results, dataset and training scripts/notes soon.

**Comment by pipipopo** [comment]
What is Isoftmax? @MartinDione
